MapboxNavigation/RouteStepFormatter.swift

```diff
@@ -6,10 +6,16 @@ import OSRMTextInstructions
 public class RouteStepFormatter: Formatter {
     let instructions = OSRMInstructionFormatter(version: "v5")
     
+    /**
+     Return an instruction as a `String`.
+     */
     public override func string(for obj: Any?) -> String? {
         return string(for: obj, markUpWithSSML: false)
     }
     
+    /**
+     Returns an instruction as a `String`. `markUpWithSSML` will return a string containing [SSML](https://www.w3.org/TR/speech-synthesis/) tag information around appropriate strings.
```

Comment:
`markUpWithSSML` is a Boolean argument; it doesn’t return anything. I think you mean that it’ll cause _the method_ to return a string containing SSML.